site/content/en/latest/tasks/extensibility/envoy-patch-policy.md

```diff
@@ -82,6 +82,34 @@ data:
+Because [EnvoyPatchPolicy][] relies on specific xDS resource names, it’s important to use the correct naming format when authoring a patch policy.
+
+| Component            | Scheme Version | Format Description                                                          | Example                          |
+|----------------------|----------------|------------------------------------------------------------------------------|----------------------------------|
+| **Listener name**    | Old            | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/http`               |
+|                      | V2             | `<Protocol>-<Port>`                                                         | `tcp-80`                         |
+| **RouteConfig name** | Old            | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/http`               |
+|                      | V2 (HTTP)      | `http-<Port>`                                                               | `http-80`                       |
+|                      | V2 (HTTPS)     | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/https`               |
+| **FilterChain name** | Old            | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/http`               |
+|                      | V2 (HTTP)      | `http-<Port>`                                                               | `http-80`                        |
+|                      | V2 (HTTPS)     | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>`                    | `default/eg/https`               |
+| **VirtualHost name** | Old            | `<GatewayNamespace>/<GatewayName>/<GatewayListenerName>/<VirtualHost>`      | `default/eg/http/www_example_com` |
+|                      | V2             | `<VirtualHost>`                                                             | `www_example_com`               |
+| **HCM StatPrefix**   | Old            | `<ApplicationProtocol>/<ContainerPort>`                                     | `http-10080`, `https-10443`     |
+|                      | V2 (HTTP)      | `http-<Port>`                                                               | `http-80`                        |
+|                      | V2 (HTTPS)     | `https-<Port>`                                                              | `https-443`                      |
+
+
+This change is gated by the XDSNameSchemeV2 runtime flag. The flag is disabled by default in v1.5 and will be enabled by default starting in v1.6.
+
+We recommend users begin migrating their [EnvoyPatchPolicy][] resources to use the version 2 naming scheme before upgrading to v1.6.
+
+To opt in to the new naming scheme early, add the`XDSNameSchemeV2` runtime flag to the `runtimeFlags.enabled` field in your [EnvoyGateway][] configuration.
```

Comment:
can we be more explicit here like https://gateway.envoyproxy.io/docs/tasks/traffic/global-rate-limit/#enable-global-rate-limit-in-envoy-gateway